source/rfc/logs.rst

```diff
@@ -1,3 +1,52 @@
 Logs
 ====
 
+Problem
+-------
+
+Our log server is full way too often. We're currently running on a 14-day policy, even though we should be keeping at least 30 days of logs.
+
+Current setup
+~~~~~~~~~~~~~
+
+We run on top of our Hetzner hypervisor with:
+
+- 4 cores
+- 8GB of RAM
+- 400GB of disk (LVM)
+
+Notes:
+
+- there's only 600GB free on the hypervisor
+- 2nd hypervisor has 800GB free
+- we seem to be generating from 20 to 40GB of logs per day (as of 2018-05-04)
+
+Retention policy
+----------------
+
+We should keep 30 days of logs for reviews - that should be enough to be able to merge properly a review.
+
+Nightlies
+~~~~~~~~~
+
+Nightly logs should follow the very same policy that build artifacts (packages and containers) will follow to ensure that we have all information we need historically. What's more, we should consider keeping nightly logs for a longer period of time (60 or 90 days), as someone might use an older build (even though we don't provide any support for them).
```

Comment:
> as someone might use an older build (even though we don't provide any support for them).

I think we could keep only parts of the logs in this case - e.g. sanity results

Reply:
We don't store sanity results on our end. Also, the size of those logs would be really minimal (especially compared to test logs), as we only have build, container and publishing logs there.

What is worth considering IMO is how to handle the logs from builds marked as monthly, as we need to keep the last 12 (full year).